src/backend/cpu/kernel/regions.hpp

```diff
@@ -178,6 +178,9 @@ void regions(Array<T> out, const Array<char> in, af_connectivity connectivity)
         }
     }
 
+    for (auto& mapElem: lmap)
+        delete mapElem.second;
```

Comment:
unique ptrs aren't abstractions. They are there to make this type of thing from not happening again. You should almost never use raw pointers